packages/esm-active-visits-app/src/index.ts

```diff
@@ -0,0 +1,36 @@
+import { defineConfigSchema, getAsyncLifecycle } from '@openmrs/esm-framework';
+import { configSchema } from './config-schema';
+
+const importTranslation = require.context('../translations', false, /.json$/, 'lazy');
+
+const backendDependencies = {
+  'webservices.rest': '^2.2.0',
```

Comment:
Are these dependencies right?

Reply:
Where can I confirm it from?

I checked from the patient chart, the dependecies were alright.

Reply:
These dependencies indicate what backend APIs you use. Which ones are you using?

Reply:
I'm using the REST API.

Reply:
Well, then remove fhir?

Reply:
Sure.